### PR TITLE
R-lesson: use zenodo link for input dataset and import to R by url

### PR DESCRIPTION
**Pull request by shiltemann**
since this works the same for both Rstudio in Galaxy and in Cloud

ping @erasche

### CODE REVIEW

**Comment by bgruening** [comment]
Will merge later, sleep well and good luck tomorrow!

File: topics/transcriptomics/tutorials/rna-seq-counts-to-viz-in-r/tutorial.md [hunk 1]
```diff
@@ -173,7 +187,7 @@ As you can see, there is a problem with how the data has been loaded. The table
 > 1. Read the file given it's a tabular file
 >
 >    ```R
->    annotatedDEgenes <- read.csv("annotatedDEgenes.tabular", sep = "\t")
+>    annotatedDEgenes <- read.csv("https://zenodo.org/record/3477564/files/annotatedDEgenes.tabular", sep = "\t")
```
Comment (bgruening) [inline topics/transcriptomics/tutorials/rna-seq-counts-to-viz-in-r/tutorial.md]: hehe like that!

**Comment by erasche** [comment]
ok, thanks @bgruening

File: topics/transcriptomics/tutorials/rna-seq-counts-to-viz-in-r/tutorial.md (hunk 1 above)
Comment (erasche) [reply topics/transcriptomics/tutorials/rna-seq-counts-to-viz-in-r/tutorial.md]: R is the best! :tada:

**Pull request merged**